src/dev/build/tasks/os_packages/create_os_package_tasks.ts

```diff
@@ -137,6 +137,7 @@ export const CreateDockerContexts: Task = {
     });
     await runDockerGenerator(config, log, build, {
       cloud: true,
+      ubuntu: true,
```

Comment:
I feel like we should change the property to something like `baseImage: 'ubi' | 'ubuntu'` so that we can't accidentally include an OS, does that sound reasonable? Maybe we could include `'none'` as an option if we specifically want that for some reason.

Reply:
Pushed with bcf9aa29f2f7cd64b39bb08144d40bdca4f3189d + 15197c3c70c5090b87d62b526f92ded58f4f234e.